ote/src/clj/ote/services/taxi_index.clj

```diff
@@ -0,0 +1,125 @@
+                  ; Fintraffic primary font
+                  {:href "https://fonts.googleapis.com/css?family=Public+Sans:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900" :rel "stylesheet"}
+                  ; Fintraffic secondary font
+                  {:href "https://fonts.googleapis.com/css2?family=Piazzolla:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" :rel "stylesheet"}])
+
+(defn ote-js-location [dev-mode?]
+  (str "js/ote"
+       (when-not dev-mode?
+         (str "-" (:current-revision-sha (current-revision-sha))))
+       ".js"))
+
+(def favicons
+  [{:rel "apple-touch-icon" :sizes "180x180" :href "/favicon/apple-touch-icon.png?v=E6jNQXq6yK"}
+   {:rel "icon" :type "image/png" :sizes "32x32" :href "/favicon/favicon-32x32.png?v=E6jNQXq6yK"}
+   {:rel "icon" :type "image/png" :sizes "16x16" :href "/favicon/favicon-16x16.png?v=E6jNQXq6yK"}
+   {:rel "manifest" :href "/favicon/manifest.json?v=E6jNQXq6yK"}
+   {:rel "mask-icon" :href "/favicon/safari-pinned-tab.svg?v=E6jNQXq6yK" :color "#5bbad5"}
+   {:rel "shortcut icon" :href "/favicon/favicon.ico?v=E6jNQXq6yK"}])
+
+(def dev-favicons
+  [{:rel "icon" :type "image/png" :sizes "32x32" :href "/favicon/favicon-dev-32x32.png"}
+   {:rel "icon" :type "image/png" :sizes "16x16" :href "/favicon/favicon-dev-16x16.png"}])
+
+(defn translations [language]
+  [:script#ote-translations {:type "x-ote-translations"}
+   (transit/clj->transit
+     {:language language
+      :translations (localization/translations language)})])
+
+(defn user-info [db user]
+  [:script#ote-user-info {:type "x-ote-user-info"}
+   (transit/clj->transit
+     (when user
+       (transport-operator/get-user-transport-operators-with-services db (:groups user) (:user user))))])
+
+(defn index-page [db user config]
+  (let [dev-mode? (:dev-mode? config)
+        testing-env? (:testing-env? config)
+        flags (str/join "," (map name (:enabled-features config)))
+        matomo-config (:matomo config)]
+    [:html
+     [:head
+      (if testing-env?
+        (for [f dev-favicons]
+          [:link f])
+        (for [f favicons]
+          [:link f]))
+      [:meta {:name "theme-color" :content "#ffffff"}]
+      [:meta {:name    "viewport"
+              :content "width=device-width, initial-scale=1.0"}]
+      [:title "FINAP"]
+      (for [{:keys [href integrity]} stylesheets]
+        [:link (merge {:rel  "stylesheet"
+                       :href href}
+                      (when (str/starts-with? href "https://")
+                        {:crossorigin ""})
+                      (when integrity
+                        {:integrity integrity}))])
+      [:style {:id "_stylefy-constant-styles_"} ""]
+      [:style {:id "_stylefy-styles_"}]
+      (translations localization/*language*)
```

Comment:
Is there need for `matomo-analytics-scripts` here?
```
      (when (not (true? dev-mode?))
        (matomo-analytics-scripts matomo-config))
```

Reply:
As mentioned in the opening comment, these are the copy-pasted parts which'll be refactored away probably soon anyways. It's kinda hard at this point to justify which element specifically still needs to exist, there's several old references to Google Analytics, Matomo, all kinds of scrollbar hacks and whatnot in these parts and it'll probably require asking from customer what the actual deal with all of them is and should they be cleared entirely.

I don't want to touch the configs itself, it's part of the Ansible deployment _(see all *.j2 files)_ and those'll break if I go modifying anything in there at this point.